[2020-11-13 18:39:21,292: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@broker:5672//: [Errno 111] Connection refused.
Trying again in 2.00 seconds... (1/100)

[2020-11-13 18:39:23,309: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@broker:5672//: [Errno 111] Connection refused.
Trying again in 4.00 seconds... (2/100)

[2020-11-13 18:39:27,339: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@broker:5672//: [Errno 111] Connection refused.
Trying again in 6.00 seconds... (3/100)

[2020-11-13 18:39:33,360: INFO/MainProcess] Connected to amqp://guest:**@broker:5672//
[2020-11-13 18:39:33,379: INFO/MainProcess] mingle: searching for neighbors
[2020-11-13 18:39:34,428: INFO/MainProcess] mingle: all alone
[2020-11-13 18:39:34,453: WARNING/MainProcess] /usr/local/lib/python3.8/site-packages/celery/fixups/django.py:205: UserWarning: Using settings.DEBUG leads to a memory
            leak, never use this setting in production environments!
  warnings.warn('''Using settings.DEBUG leads to a memory
[2020-11-13 18:39:34,454: INFO/MainProcess] celery@6ef45dc24859 ready.
[2020-11-13 18:49:30,229: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@broker:5672//: [Errno 111] Connection refused.
Trying again in 2.00 seconds... (1/100)

[2020-11-13 18:49:32,271: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@broker:5672//: [Errno 111] Connection refused.
Trying again in 4.00 seconds... (2/100)

[2020-11-13 18:49:36,291: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@broker:5672//: [Errno 111] Connection refused.
Trying again in 6.00 seconds... (3/100)

[2020-11-13 18:49:42,312: INFO/MainProcess] Connected to amqp://guest:**@broker:5672//
[2020-11-13 18:49:42,330: INFO/MainProcess] mingle: searching for neighbors
[2020-11-13 18:49:43,395: INFO/MainProcess] mingle: all alone
[2020-11-13 18:49:43,420: WARNING/MainProcess] /usr/local/lib/python3.8/site-packages/celery/fixups/django.py:205: UserWarning: Using settings.DEBUG leads to a memory
            leak, never use this setting in production environments!
  warnings.warn('''Using settings.DEBUG leads to a memory
[2020-11-13 18:49:43,421: INFO/MainProcess] celery@6ef45dc24859 ready.
